rio/srcrobot/commands/TurnInPlace.py

```diff
@@ -23,8 +24,12 @@ def initialize(self):
         self.turnPID.setSetpoint(self.angle)
 
     def getRotationValue(self):
-        self.angularvelMRadiansPerSecond = self.turnPID.calculate(self.s_Swerve.getGyroYaw().radians())
-        return self.angularvelMRadiansPerSecond
+        rotationStick = self.currentRotation()
+        if abs(rotationStick) > 0.0:
+            return rotationStick
+        else:
+            self.angularvelMRadiansPerSecond = self.turnPID.calculate(self.s_Swerve.getGyroYaw().radians())
+            return self.angularvelMRadiansPerSecond
```

Comment:
Should this be using heading rather than yaw?

Reply:
Yes it should, this hasnt really been touched in a while.